README.md

```diff
@@ -62,10 +62,32 @@ The distributed sessions module can be configured through the environment variab
 | session.DatastoreStore.sessionKind  |  Name of the entity used to store sessions in the Datastore. |  TomcatGCloudSession |
 | session.DatastoreStore.sessionMaxInactiveTime |  Defines the maximum time (in seconds) a session can be inactive before being deleted by the expiration process. | 3600 |
 
-## Running outside of Google Cloud
-If you are using the runtime outside of Google Cloud, you will want to make sure that your application has access to
+## Usage outside of Google Cloud Platform
+If you are using the runtime outside of GCP, you will want to make sure that your application has access to
 the Datastore. In this case, check out the [Google Cloud Authentication](https://developers.google.com/identity/protocols/application-default-credentials) guide.
 
+# Stackdriver Trace
+The trace module sends information about requests (such as latency) to the [Stackdriver Trace service](https://cloud.google.com/trace/docs/).
+
+To enable this module add `trace-requests` to the list of enabled modules.
+
+```yaml
+env_variables:
+  TOMCAT_MODULES_ENABLE: trace-requests
```

Comment:
I'm having some difficulty with the name of the module being `trace-requests`. Maybe rename it to just `trace` or `stackdriver-trace`. We may extend it in the future to support tracing GRPC requests triggered from the request worker. So, a less specific name is probably better.